apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -79,22 +81,13 @@ export class GetNotificationsFeed {
         seen: command.query.seen,
         read: command.query.read,
       },
-      { limit: COUNT_LIMIT }
-      /*
-       * todo NV-2161 in version 0.16
-       *  update option as below,
-       *  update below:  hasMore = feed.length < totalCount
-       *  remove totalCount
-       * { skip: command.page * command.limit, limit: command.limit + 1 }
-       */
+      { limit: COUNT_LIMIT, skip }
```

Comment:
small question, I think we have here a small loophole,
if the user will provide a `limit` bigger than 100 for example 1000 (which is allowed, I don't fount any limitation to it) then:
* The total count won't make sense if the feed length is 900 but the total count is 100.
* The hasMore will be falsy, if the feed length will be 1000 but the total count will be 100, the hasMore will be false but the user could have more notifications. 

Let me know what you think, maybe a missed something.

Reply:
Ah yes, should we limit the limit to 100 or do you have a suggestion on how we could solve this?

Reply:
It could be a possible solution i am just not sure regarding the product part of this change, `should we support a higher limit or not`.
Maybe we could do it optional to update to 100/1000 (but so they both limit and count limit would overlap)

Reply:
so use `command.limit` instead of `COUNT_LIMIT`?

Reply:
Generally yes 🙃, but I wonder if it is ok, product-wise. wdyt?

Reply:
as long as we have skip as well in the query I do not see any problem with it...